db/migrate/20230810120936_create_reports.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReports < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reports do |t|
+      t.belongs_to :user, null: false, foreign_key: true
+      t.string :title,    null: false
+      t.text :content
```

Comment:
インデントが中途半端に揃えられています。以下のどちらかにしましょう。

```suggestion
      t.belongs_to :user,    null: false, foreign_key: true
      t.string     :title,   null: false
      t.text       :content
```

```suggestion
      t.belongs_to :user, null: false, foreign_key: true
      t.string :title, null: false
      t.text :content
```

Reply:
すみません、確認不足でした🙇‍♀️
修正しました。https://github.com/SuzukaHori/fjord-books_app-2023/pull/7/commits/2329c440d54b9dd75587d307c7d918629f6acedd